### PR TITLE
migrate over communications & outreach page (with updates)

### PR DESCRIPTION
**Pull request by Aariq**
Partly addresses #7

### CODE REVIEW

File: outreach-communication.qmd [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+    -   [nmcisaac\@arizona.edu](mailto:nmcisaac@arizona.edu) Naomi McIsaac for Environmental Science
+
+    -   [prupert\@arizona.edu](mailto:prupert@arizona.edu) Peggy Rupert for Nutritional Sciences and Wellness
+
+    -   [pnielsen322\@arizona.edu](mailto:pnielsen322@arizona.edu) Paula Nielsen for Entomology
+
+    -   [thodges\@arizona.edu](mailto:thodges@arizona.edu) Tanya Hodges for AETI (Yuma)
+
+-   Data Science Institute ["Data Drip"](https://bcf.arl.arizona.edu/news) at [Rii-datascienceinstitute\@arizona.edu](mailto:Rii-datascienceinstitute@arizona.edu) <!--# this URL is broken.  What is "Data Drip" and does it exist still? -->
```
Comment (Chris-Schnaufer) [inline outreach-communication.qmd:49]: I checked out the `https://bcf.arl.arizona.edu/news` link (aka: data drip news) and it looks good. Is that the URL that was broken? **Data Drip** appears to be what they call the monthly news letter

File: outreach-communication.qmd [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+-   R-Users AZ listserv, send announcement to Jeff Oliver [jcoliver\@email.arizona.edu](mailto:jcoliver@email.arizona.edu)
+
+-   CALS Data Science Ambassadors
```
Comment (Chris-Schnaufer) [inline outreach-communication.qmd:53]: URL https://datascience.arizona.edu/dsa?

File: outreach-communication.qmd (hunk 1 above)
Comment (Aariq) [reply outreach-communication.qmd:49]: Huh, might have been a temporary issue

File: outreach-communication.qmd [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+#### Guidelines
+
+-   Please review the group [code of conduct](code-of-conduct.qmd) before posting to ensure you adhere to these guidelines
+
+-   Including relevant pictures in posts are encouraged.
+    **Please get explicit verbal/written consent from everyone in the picture before posting.**
+
+-   Cite sources and mention them (\@journal or \@author) if possible
+
+-   If anyone comments on a post, make sure to respond to them, especially if it's a question.
```
Comment (Chris-Schnaufer) [inline outreach-communication.qmd:167]: Perhaps add `Responses can include emojis, comments, and replies. If responding out of channel, consider replying with an emoji in channel`?

File: outreach-communication.qmd [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+-   Example hashtags: #UArizona, #UniversityOfArizona, #Rstats, #DataScience, #DigitalAg, #DataViz, #AcademicChatter, #RSEng
+
+#### Ideas for posts
+
+-   Feel free to get creative!
+    This is just a limited list of possible selections.
+
+-   Content should be directly in line with our mission statement
+
+    -   **The mission of the UA ag data group is to provide scientists and engineers with open software, data, and computing that will allow more efficient discovery and invention so that we can engineer crops and manage sustainable agricultural landscapes that produce food, energy, and ecosystem services.**
```
Comment (Chris-Schnaufer) [inline outreach-communication.qmd:181]: Change to a link to our current mission statement (undergoing revision as I write this)

File: outreach-communication.qmd [hunk 5]
```diff
@@ -0,0 +1,251 @@
+---
+title: "Communications and Outreach"
+---
+
+## Documents and Websites to Update regularly
+
+-   [CCT Data Science website](https://datascience.cct.arizona.edu/)
+
+-   [CCT Service Catalog](https://docs.google.com/document/d/1LIxkktiP219_Xd24vXpgA1Qz7j7WcuRORtN26p2Qujw/edit)
```
Comment (Aariq) [inline outreach-communication.qmd:9]: I have literally never seen this before.  Do we need to review and update it?

File: outreach-communication.qmd [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+
+    -   [pnielsen322\@arizona.edu](mailto:pnielsen322@arizona.edu) Paula Nielsen for Entomology
+
+    -   [thodges\@arizona.edu](mailto:thodges@arizona.edu) Tanya Hodges for AETI (Yuma)
+
+-   Data Science Institute ["Data Drip"](https://bcf.arl.arizona.edu/news) at [Rii-datascienceinstitute\@arizona.edu](mailto:Rii-datascienceinstitute@arizona.edu)
+
+-   R-Users AZ listserv, send announcement to Jeff Oliver [jcoliver\@email.arizona.edu](mailto:jcoliver@email.arizona.edu)
+
+-   CALS [Data Science Ambassadors](https://datascience.arizona.edu/dsa)
```
Comment (Aariq) [inline outreach-communication.qmd:53]:
```suggestion
-   CALES [Data Science Ambassadors](https://datascience.arizona.edu/dsa)
```

File: outreach-communication.qmd [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+To log in, you'll find the password on [stache](https://stache.arizona.edu/) (you may need to ask for it to be shared with you if you don't see it)
+
+#### Guidelines
+
+-   Please review the group [code of conduct](code-of-conduct.qmd) before posting to ensure you adhere to these guidelines
+
+-   Including relevant pictures in posts are encouraged.
+    **Please get explicit verbal/written consent from everyone in the picture before posting.**
+
+-   Cite sources and mention them (\@journal or \@author) if possible
```
[the remark this answers is not in the document]
Comment (Aariq) [reply outreach-communication.qmd:159]:
```suggestion
-   Cite sources and mention users (\@journal or \@author) in posts when applicable
```

File: outreach-communication.qmd [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+
+We have a mastodon account [\@cct_datascience\@fosstodon.org](https://fosstodon.org/@cct_datascience).
+To log in, you'll find the password on [stache](https://stache.arizona.edu/) (you may need to ask for it to be shared with you if you don't see it)
+
+#### Guidelines
+
+-   Please review the group [code of conduct](code-of-conduct.qmd) before posting to ensure you adhere to these guidelines
+
+-   Including relevant pictures in posts are encouraged.
+    **Please get explicit verbal/written consent from everyone in the picture before posting.**
```
Comment (Aariq) [inline outreach-communication.qmd:157]:
```suggestion
-   Including relevant pictures in posts are encouraged.
    **Please get explicit verbal/written consent from everyone in the picture before posting.**
    Tag people in the pictures who are on Mastodon **with their permission**
```

File: outreach-communication.qmd (hunk 3 above)
Comment (Aariq) [reply outreach-communication.qmd:161]: I didn't quite understand this comment.  I'm going to go ahead and merge this PR, but please feel free to open an issue with more details on this.

**Pull request merged**